qaframework-bdd-tests/src/test/resources/features/refapp-2.x/09-form/forms.feature

```diff
@@ -6,7 +6,8 @@ Feature: Form Management
   @selenium
   @form
   Scenario: Form management
-   When  a user check the availability of forms
+   Then	the system loads manage Html Forms Page
```

Comment:
I think the better approach is to have this step within the background. @sherrif10 what do you think?